hack/verify-govet.sh

```diff
@@ -30,7 +30,7 @@ function govet_blacklist_contains() {
 }
 
 for test_dir in $(os::util::list_go_src_dirs); do
-	if ! result="$(go tool vet -shadow=false "${test_dir}" 2>&1)"; then
+	if ! result="$(go tool vet -shadow=false -printfuncs=Info,Infof,Warning,Warningf "${test_dir}" 2>&1)"; then
```

Comment:
i'm going on record as liking it better the way it was.

Reply:
And expecting that people understand this this _one_ place the "${args}" are _not_ quoted because we _need_ word splitting for proper parameter passing for the call to `go tool vet`? No thanks